### PR TITLE
ocm: add insights client

### PR DESCRIPTION
**Pull request by jlpadilla**
Onboard project open-cluster-management/insights-client

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/insights-client/OWNERS [hunk 1]
```diff
@@ -0,0 +1,20 @@
+approvers:
+  - jlpadilla
+  - xrajesh
+  - SherinV
+  - mqhackett
+  - arewm
```
Comment (mqhackett) [inline ci-operator/jobs/open-cluster-management/insights-client/OWNERS:6]:
```suggestion
  - zlayne
  - arewm
```

File: ci-operator/jobs/open-cluster-management/insights-client/OWNERS [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - ray-harris
+  - schmidtd
+  - smithbw88
+  - tpouyer
+reviewers:
+  - jlpadilla
+  - xrajesh
+  - SherinV
+  - mqhackett
+  - arewm
```
Comment (mqhackett) [inline ci-operator/jobs/open-cluster-management/insights-client/OWNERS:16]:
```suggestion
  - zlayne
  - arewm
```

File: ci-operator/config/open-cluster-management/insights-client/OWNERS [hunk 3]
```diff
@@ -0,0 +1,20 @@
+approvers:
+  - jlpadilla
+  - xrajesh
+  - SherinV
+  - mqhackett
+  - arewm
```
Comment (mqhackett) [inline ci-operator/config/open-cluster-management/insights-client/OWNERS:6]:
```suggestion
  - zlayne
  - arewm
```

File: ci-operator/config/open-cluster-management/insights-client/OWNERS [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  - ray-harris
+  - schmidtd
+  - smithbw88
+  - tpouyer
+reviewers:
+  - jlpadilla
+  - xrajesh
+  - SherinV
+  - mqhackett
+  - arewm
```
Comment (mqhackett) [inline ci-operator/config/open-cluster-management/insights-client/OWNERS:16]:
```suggestion
  - zlayne
  - arewm
```

File: ci-operator/config/open-cluster-management/insights-client/OWNERS [hunk 5]
```diff
@@ -0,0 +1,20 @@
+approvers:
```
Comment (mqhackett) [inline ci-operator/config/open-cluster-management/insights-client/OWNERS:1]: The owners file will get synced with the upstream owners file. We should make these the same:
https://github.com/open-cluster-management/insights-client/blob/main/OWNERS#L5

Comment (jlpadilla) [reply ci-operator/config/open-cluster-management/insights-client/OWNERS:1]: updated

**Comment by jlpadilla** [comment]
/assign @petr-muller

This PR is ready for review and merge.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**